lib/omnibus/packagers/msi.rb

```diff
@@ -73,9 +76,13 @@ class Packager::MSI < Packager::Base
             -cultures:en-us
             -loc "#{windows_safe_path(staging_dir, 'localization-en-us.wxl')}"
             project-files.wixobj source.wixobj
-            -out "#{windows_safe_path(Config.package_dir, package_name)}"
+            -out "#{msi_file}"
         EOH
         shellout!(light_command, returns: [0, 204])
+
+        if cert_name
```

Comment:
I'm not seeing this in RPM:
https://github.com/chef/omnibus/blob/master/lib/omnibus/packagers/rpm.rb#L307
https://github.com/chef/omnibus/blob/master/lib/omnibus/packagers/rpm.rb#L90-L96